CodeU_Assignement_1/src/codeu_assignement_1/main.java

```diff
@@ -0,0 +1,278 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package codeu_assignement_1;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+
+/**
+ *
+ * @author Karine
+ */
+public class main {
+    /**
+     * @param args the command line arguments
+     * 
+     * Gives a basic interface to do some partial testing to the code
+     * 
+     */
+    public static void main(String[] args) {
+        if (args.length == 0) // run tests!
+        {
+            run_tests_Q1();
+            run_tests_Q2();
+            return;
+        }
+        
+        // TODO code application logic here

```

Comment:
Old code, oups... I work a lot with TODO to get the general structure of what will be later there...